code/modules/mod/mod_theme.dm

```diff
@@ -626,14 +612,14 @@
 	armor = list(MELEE = 30, BULLET = 30, LASER = 5, ENERGY = 5, BOMB = INFINITY, RAD = 75, FIRE = 75, ACID = 150) //Slow balistic / explosive testing armor. Not laser testing however!
 
 /datum/mod_theme/security
-	name = "security"
-	desc = "A Shellguard Munitions security suit, offering quicker speed at the cost of carrying capacity."
-	extended_desc = "A Shellguard Munitions classic, this model of MODsuit has been designed for quick response to \
-		hostile situations. These suits have been layered with plating worthy enough for fires or corrosive environments, \
-		and come with composite cushioning and an advanced honeycomb structure underneath the hull to ensure protection \
-		against broken bones or possible avulsions. The suit's legs have been given more rugged actuators, \
-		allowing the suit to do more work in carrying the weight. However, the systems used in these suits are more than \
-		a few years out of date, leading to an overall lower capacity for modules."
+	name = "'Takabara' security"
+	desc = "A last-gen Shellguard Munitions combat modsuit intended for corporate security forces. Decently armored and highly mobile, but somewhat lackluster in modification potential."
```

Comment:
I don't know if last-gen needs to be included in the base description, since it might make players think it's weaker than it is? Might be better suited moving that tidbit to the extended description.